projects/valtimo/config/assets/core/en.json

```diff
@@ -920,5 +920,32 @@
     "enum": "Enumeration",
     "arrayCount": "Count",
     "underscoresToSpaces": "Underscores to spaces"
+  },
+  "objectManagement": {
+    "title": "Objecttypes",
+    "createObjecttype": "Create new objecttype",
+    "labels": {
+      "title": "Title",
+      "objects": "Objects API plugin configuration",
+      "objecttypes": "Objecttypen API plugin configuration",
+      "objecttypeUrl": "Objecttype URL",
+      "formDefinitionView": "View form definition",
+      "formDefinitionEdit": "Edit form definition"
+    },
+    "tooltips": {
+      "title": "Title",
+      "objects": "Objects API plugin configuration",
+      "objecttypes": "Objecttypen API plugin configuration",
+      "objecttypeUrl": "Objecttype URL",
+      "formDefinitionView": "View form definition",
+      "formDefinitionEdit": "Edit form definition"
```

Comment:
Change to:
The name of the form IO definition that will be used to edit the object.